QuizApp/Scenes/LoginPage/ViewController/LoginViewController.swift

```diff
@@ -9,15 +9,143 @@ import UIKit
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = .loginPageIllustration
+        return imageView
+    }()
+    
+    private let loginStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = Sizing.stackViewSpacing
+        stackView.alignment = .center
+        return stackView
+    }()
+    
+    private let input: UITextField = {
+        let input = UITextField()
+        input.isUserInteractionEnabled = true
+        input.textColor = .neutralGrey
+        input.textAlignment = .center
+        let placeholderText = "შეიყვანეთ სახელი"
+        let attributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont.systemFont(ofSize: FontSizes.xs)
+        ]
+        input.attributedPlaceholder = NSAttributedString(string: placeholderText, attributes: attributes)
+        input.layer.borderWidth = Sizing.inputBorder
+        input.layer.borderColor = Colors.border.cgColor
+        input.layer.cornerRadius = Sizing.inputRadius
+        return input
+    }()
     
+    private let startButton = YellowRoundedButton(
+        title: "ქვიზის დაწყება",
+        width: Sizing.startButtonWidth,
+        height: Sizing.startButtonHeight,
+        radius: Sizing.startButtoRadius,
+        fontSize: FontSizes.xs
+    )
     
+    // MARK: - Constraints
+    private var blueBackgroundTopConstraint: NSLayoutConstraint!
+    
+    // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
+        setupUI()
+        setupKeyboardObservers()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        removeKeyboardObservers()
+    }
+    
+    // MARK: - UI Setup
+    private func setupUI() {
+        setupView()
+        setupViewHierarchy()
+        setConstraints()
+    }
+    
+    private func setupView() {
+        view.backgroundColor = .white
+    }
+    
+    private func setupViewHierarchy() {
+        loginStackView.addArrangedSubviews(input, startButton)
+        view.addSubviews(blueBackground, titleLabel, illustration, loginStackView)
+        view.bringSubviewToFront(titleLabel)
+    }
+    
+    private func setConstraints() {
+        blueBackgroundTopConstraint = blueBackground.topAnchor.constraint(equalTo: view.topAnchor, constant: Sizing.blueBackgroundTopAnchor)
+        
+        NSLayoutConstraint.activate([
+            blueBackgroundTopConstraint,
+            blueBackground.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Sizing.blueBackgroundLeadingAnchor),
+            blueBackground.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: Sizing.blueBackgroundTrailingAnchor),
+            blueBackground.heightAnchor.constraint(equalToConstant: Sizing.blueBackgroundHeight),
```

Comment:
do you need assigning height here? it might be a problem for smaller device like SE